sound/soc/intel/boards/sdw_rt711_rt1308_rt715.c

```diff
@@ -789,7 +792,14 @@ static int sof_card_dai_links_create(struct device *dev,
 #endif
 
 	ssp_mask = SOF_SSP_GET_PORT(sof_rt711_rt1308_rt715_quirk);
-	ssp_num = hweight_long(ssp_mask);
+	/*
+	 * on generic tgl platform, i2s or sdw mode is supported
+	 * based on board rework. A acpi device is registered in
```

Comment:
"acpi" isn't a commonly accepted style IMHO

Reply:
I search acpi & ACPI,  i2s & I2S in git log. yes, ACPI & I2S is more popular. Thanks! I will refine it